app/src/main/java/com/keylesspalace/tusky/entity/TrendsLink.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2023 Tusky Contributors
+ *
+ * This file is a part of Tusky.
+ *
+ * This program is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Tusky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+ * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Tusky; if not,
+ * see <http://www.gnu.org/licenses>.
+ */
+
+package com.keylesspalace.tusky.entity
+
+import com.google.gson.annotations.SerializedName
+
+enum class PreviewCardKind {
+    @SerializedName("link")
+    LINK,
+
+    @SerializedName("photo")
+    PHOTO,
+
+    @SerializedName("video")
+    VIDEO,
+
+    @SerializedName("rich")
+    RICH
+}
+
+/**
+ * Representation of a Mastodon [PreviewCard](https://docs.joinmastodon.org/entities/PreviewCard/)
+ */
+interface PreviewCard {
+    val url: String
+    val title: String
+    val description: String
+    val kind: PreviewCardKind
+    val authorName: String
+    val authorUrl: String
+    val providerName: String
+    val providerUrl: String
+    val html: String
+    val width: Int
+    val height: Int
+    val image: String?
+    val embedUrl: String
+    val blurhash: String?
+}
+
+data class LinkHistory(
+    val day: String,
+    val accounts: Int,
+    val uses: Int
+)
+
+data class TrendsLink(
```

Comment:
What is a trends link (or a trending link in other places)?

(Maybe small comment.)

Reply:
Done.

Reply:
It says there that it is a "trends link" - but no further explanation.